Backend/router/v1/user.js

```diff
@@ -32,14 +38,18 @@ router.post('/signup', (req, res) => {
     }
   }).then(result => {
     if (result == "") {
-      models.user.create({
-        email: req.body.email,
-        password: hashedPassword,
-      }).then(result => {
-          res.send(
-            "< " + req.body.email + " > membership has been completed."
-          );
-      });
+      if (password_length_check(req.body.password.length) == false ) {
```

Comment:
Can you do !password_length_check(req.body.password.length) here? Maybe I'm confused because I'm using Java, c++ these days.